src/component/Person/Person.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => {
+  const partnerSex = (sex === 'm')
+    ? 'wife'
+    : 'husband';
+
+  const personParnter = isMarried
+    ? `My ${partnerSex} name is ${partnerName}`
+    : 'No partner';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+      <p className="Person__partner">
+        {personParnter}
+      </p>
+    </section>
+  );
+};
+
+Person.propTypes = {
+  name: PropTypes.string.isRequired,
+  age: PropTypes.number.isRequired,
```

Comment:
Not everything is required here.
![image](https://user-images.githubusercontent.com/60885495/140271871-92291b83-872b-4a35-a73c-3e92d1e693fc.png)